### ISSUE TITLE
I am confused with instructions 

### ISSUE THREAD

**Issue by Thewierduser** [zdc54/nylon#1]
1. If you have no money, go ahead and use [Repl.it.] (https://replit.com)
First, go ahead and go to the proxy URLs listed above, and look in the readme.md for the "Deploy to Replit" button.
make sure you name them accordingly and memorably.

What is this asking to do?
-----------------------------------------------------------
2. Next, go into `/pages/`, and look for `womginx.html`, `alloy.html`, `palladium.html`.
Go ahead and enter the links for each proxy inside of the url placeholder box.
Now go ahead and make a new HTML Repl.
Once you are in, go to console, and type this:
`git clone https://github.com/BinBashBanana/gfiles`.
Drag the files from the gfiles folder (This might lag. If the files dissapear after dragging them to the root of the project, just reload. this is a common bug.)

Do i just put the links to the github page?

**Comment by nevr13** [comment]
no, you would link it to /gfiles/html5/ or host the other site on replit and link the iframe through there.
